terraform/projects/infra-networking-dev/main.tf

```diff
@@ -0,0 +1,69 @@
+terraform {
+  required_version = "= 0.11.10"
+
+  backend "s3" {
+    bucket = "re-observe-dev"
+    key    = "infra-networking-dev.tfstate"
```

Comment:
should this be `-modular` like the others, given we hardcode the filename (but allow the bucket name to vary)?